src/Components/ColorPicker/ColorPicker.tsx

```diff
@@ -132,6 +132,7 @@ export const ColorPicker = forwardRef<ColorPickerSwatchRef, ColorPickerProps>(
           <Input
             className="cf-color-picker--input"
             placeholder="#000000"
+            inputStyle={{paddingLeft: '29px'}}
```

Comment:
It might be better to do this in the stylesheet instead of inline using the `.cf-color-picker--input` class name.

Reply:
@mavarius okay then I'm on it